src/main/java/core/basesyntax/vehicles/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.vehicles;
+
+import core.basesyntax.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
```

Comment:
The message printed should be more informative and indicate the specific type of machine that is working. Consider including the class name in the message.